### PR TITLE
[Back 6 22] adding models for song & playlist

### PR DESCRIPTION
**Pull request by sanderpol**
This pull request applies to the following story: https://app.asana.com/0/1199140982001327/1199172305250014 and https://app.asana.com/0/1199140982001327/1199156508219597

## Description

**Changes proposed in this pull request (technical):**
* Added Song Model
* Added Playlist Model
* Added PlaylistSong Model

## Checklist:
- [x] :nerd_face: Documentation for this PR is covered.

### CODE REVIEW

File: View/MainWindow.xaml.cs [hunk 1]
```diff
@@ -20,9 +23,12 @@ namespace Soundify
...
     {
+        public DatabaseContext Context { get; set; }
+
         public MainWindow()
         {
             InitializeComponent();
+            Context = new DatabaseContext();
         }
     }
 }
```
Comment (WhatTheHai) [inline View/MainWindow.xaml.cs:34]: Should not be included in the models

**Review by WhatTheHai — CHANGES_REQUESTED** [review]

File: View/MainWindow.xaml.cs (hunk 1 above)
Comment (sanderpol) [reply View/MainWindow.xaml.cs:34]: Nope that's init of the ef Database context

Comment (WhatTheHai) [reply View/MainWindow.xaml.cs:34]: Ok!

**Review by WhatTheHai — APPROVED** [review]
JOB WELL DONE!

**Pull request merged**